appservice/package.json

```diff
@@ -35,6 +35,7 @@
         "azure-arm-resource": "^3.0.0-preview",
         "azure-arm-storage": "^3.1.0",
         "azure-arm-website": "^3.0.0-preview",
+        "fs-extra": "^4.0.2",
```

Comment:
I forgot to mention, but you need to add the license to thirdpartynotices when adding a new package https://github.com/Microsoft/vscode-azuretools/blob/master/appservice/thirdpartynotices.txt

Just follow the format used in that file